File: example/javabuilds/5-test-suite/qux/test/src/QuxTests.scala
package qux
import utest._
object QuxTests extends TestSuite {
  def tests = Tests {
    test("hello") {
      val result = Qux.hello()
      assert(result.startsWith("Hello"))
      result
    }
    test("world") {
      val result = Qux.hello()
      assert(result.endsWith("World"))
      result
    }
  }
}

File: example/javabuilds/5-test-suite/qux/test/src/QuxTests.java
package qux;

import static org.junit.Assert.assertTrue;
import org.junit.Test;

public class QuxTests {

  @Test
  public void hello() {
    String result = Qux.hello();
    assertTrue(result.startsWith("Hello"));
  }

  @Test
  public void world() {
    String result = Qux.hello();
    assertTrue(result.endsWith("World"));
  }
}